src/__tests__/utils/testForm.tsx

```diff
@@ -17,35 +27,72 @@ export function TextField(props: {
   const {
     field: { onChange, value },
   } = useController({ control: props.control, name: props.name });
+
   return (
     <div data-testid={textFieldTestId}>
       {label && <label>{label}</label>}
       <input
-        data-testid={props.testId}
+        name={props.name}
+        data-testid={props.testId || defaultTextInputTestId}
         onChange={(e) => {
           onChange(e.target.value);
         }}
         value={value ? value : ""}
         placeholder={placeholder}
       />
+      <ErrorMessage />
     </div>
   );
 }
 
-function BooleanField(props: {
+export function BooleanField(props: {
```

Comment:
just made these test fields a little more robust and exported them, didn't break any previous tests